README.md

```diff
@@ -30,15 +29,19 @@
 
 # Running DB and App
 
-1. Start dev server in another terminal window then nagivate in a new browser tab to `localhost:3000`.
+1. Start dev server in another terminal window then nagivate in a new browser 
```

Comment:
```suggestion
1. Start the dev server in another terminal window, then navigate in a new browser
```

This removes a trailing space, and fixes a typo with the word `navigate`.

Reply:
thx fixed

Reply:
You're welcome! Thanks for accepting the suggestion.